presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -113,41 +142,126 @@ The configuration properties of Presto C++ workers are described here, in alphab
   worker node. Memory for system usage such as disk spilling and cache prefetch are
   not counted in it.
 
-``query-reserved-memory-gb``
+``shared-arbitrator.reserved-capacity``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-* **Type:** ``integer``
-* **Default value:** ``4``
+* **Type:** ``string``
+* **Default value:** ``4GB``
 
   Specifies the total amount of memory in GB reserved for the queries on
   a worker node. A query can only allocate from this reserved space if
   1) the non-reserved space in ``query-memory-gb`` is used up; and 2) the amount
-  it tries to get is less than ``memory-pool-reserved-capacity``.
+  it tries to get is less than ``shared-arbitrator.memory-pool-reserved-capacity``.
 
-``runtime-metrics-collection-enabled``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-* **Type:** ``boolean``
+``shared-arbitrator.memory-pool-initial-capacity``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
The formatting line (`^^^`) should have the same number of characters as the line above it.